.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build Mu
+
+on:
+  push:
+    branches: '*'
+  pull_request:
+    branches: '*'
```

Comment:
I'm not sure it makes sense to run on both push and pull_request. At least it seems that this execute all the actions twice for every pull request. Perhaps it should be:

```
  push:
    branches: 'master'
  pull_request:
    branches: '*'
```

Reply:
I think we want the test to run in all pushes to all branches, as that is useful during development before creating a PR.

But yes, we could only build the executables on master and PR 👍